pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -116,6 +116,24 @@ func (o CreateBootstrapPolicyFileOptions) CreateBootstrapPolicyFile() error {
 		policyTemplate.Objects = append(policyTemplate.Objects, versionedObject)
 	}
 
+	controllerRoles := bootstrappolicy.GetBootstrapControllerRoles()
```

Comment:
@liggitt I like the upstream, reconcile on start combined with system:masters authorized separately which obviated the need for an external `policy.json`.  You see any reason to keep creating the "BootstrapPolicyFile"?